docs/content/configuration/csi_ug_config_create_storageclasses.md

```diff
@@ -1,14 +1,14 @@
 # Creating a StorageClass
```

Comment:
how come this (and its topology version, and snapshots) filename is plural, while the rest are singular?

Reply:
the title was originally in plural. the name of the file itself is now only reflected in github. if I change it more linking between versions will break. i don't think it's critical to update

Reply:
> more linking between versions will break

could you give an example? I thought our new versioning system will prevent those breakages

Reply:
it's on the ibm docs side. it would also mean other extra work that I don't think is necessary for this release. 
for example sitting in a specific topic and wanting to see what it is between versions
![image](https://user-images.githubusercontent.com/54570654/134349259-72a2a859-8f09-4147-b889-b6a89419bb32.png)
let's leave as-is for now please

Reply:
can't we specify the rename in the ibm docs side so that it could correctly redirect?

Reply:
no. these are my source files. it's how it's picked up. I'm not starting to rename things this close to GA when it's not necessary. If you want, please open a ticket for next release and I'll do it near the beginning of the release. it won't redirect though to the exact page.

Reply:
done: CSI-3403